sla_per_user_system_test.py

```diff
@@ -195,23 +228,33 @@ def run_stress_and_verify_threads(self, params=None):
 
         return read_queue
 
+    def one_run_c_s_stats(self, read_run, user_name, statistic_name):
+        res = self.get_stress_results(queue=read_run, store_results=False)
+        stat_rate, username = None, None
+        if res:
+            stat_rate = res[0].get(statistic_name)
+            username = res[0].get('username')
+
+        if not (stat_rate and username):
+            self.log.error(f'Stress statistics are not received for user {user_name}. Can\'t complete the test')
```

Comment:
Two things to change: you can use double quotes instead of escaping single quote inside the log message and it's better to use lazy logging.

Reply:
fixed